conda-installation.md

```diff
@@ -247,3 +260,66 @@ arguments = myscript.py
 transfer_input_files = Miniconda3-latest-Linux-x86_64.sh, script.py, other_input.file
 ```
 {:.sub}
+
+# Pinning Dependencies
+
+An important part of improving reproducibility and consistency between runs
+is to ensure that you use the correct/expected versions of your dependencies.
+
+When you run a command like `conda install numpy`, `conda` tries to install
+**the most recent version of `numpy`**. For example, `numpy` version `1.18.2`
+was released on March 17, 2020. To install exactly this version of `numpy`, you
+would run `conda install numpy==1.18.2` (the same works for `pip`).
+
+`conda` provides a way to "lock" or "pin" the versions of all of your packages.
```

Comment:
I'd replace this with something like: 
If you want a record of your environment or want to reproduce it exactly somewhere else, conda can...